content/news/XXXX-XX-XX-mixxx-2-5-beta-released.md

```diff
@@ -0,0 +1,51 @@
+title: Mixxx 2.5 beta released
+authors: Evelynne Veys
+tags: 2.5, beta, release announcement
+comments: yes
+status: draft
+
+#### Dear Mixxx-ers
+
+Funny things happen in spring, birds start nesting and lay eggs, and so did the Mixxx developers.
+A bit late for Easter but it is a magnificent giant gift filled with smaller gifts: Mixxx 2.5 beta!
+You can find the details about the new version below.
+
+#### Appeal
+
+Before releasing 2.5 as the next stable release, we need it to be tested thoroughly, something we can't do all on our own.
+After some time looking at the same release and searching for solutions, we might overlook minor bugs or problems.  
+That's why we need **YOU** to help us. The more people testing this beta and reporting eventual bugs to the [bug tracker](https://github.com/mixxxdj/mixxx/issues), the faster the beta can be promoted to stable.  
+In case you're scared of messing up your neatly configured PC and losing your holy data is your Haloween nightmare, read the new wiki article [Safeguard Your Mixxx Data](https://github.com/mixxxdj/mixxx/wiki/Safeguard-your-Digital-DJ-Data).
+It explains how to creata a timestamped image, how to back up your data and offers tips and tricks.  
+A beta release contains the solutions for problems that occurred and were notified by users who tested the alpha release.
+The beta release is a general check before the new version can be released.  
+The Mixxx database and settings of Mixxx 2.4.x stable are equal to those in Mixxx 2.5 beta, which means you can install the 2.5 over the 2.4 and vice versa without a problem.  
+After you have successfully tested this 2.5 beta for yourself, you can continue using this version in your bedroom or even during live DJ-ing, and thus benefit from the new features.  
+Please [join Mixxx](https://mixxx.org/get-involved/).
+In Mixxx, we (can) trust.
+
+#### Highlights
+
+* In version 2.5, Mixxx moves from depending on Qt 5 to [Qt 6](https://www.qt.io/product/qt6). This will improve hardware support and will enable Mixxx to be compatible with latest operating systems and hardware.
```

Comment:
```suggestion
* In version 2.5, Mixxx moves from depending on Qt 5 to [Qt 6](https://www.qt.io/product/qt6). This will improve hardware support and will enable Mixxx to be compatible with latest operating systems and hardware. However, this upgrade necessitates the discontinuation of support for older operating systems, specifically macOS versions earlier than 11, Windows 7 and 8.1 and Ubuntu 20.04 "Focal Fossa" and older.
```
```